src/txrequest.cpp

```diff
@@ -306,23 +306,28 @@ GenTxid ToGenTxid(const Announcement& ann)
 }  // namespace
 
 /** Actual implementation for TxRequestTracker's data structure. */
-class TxRequestTracker::Impl {
+class TxRequestTracker::Impl
+{
+private:
+    mutable Mutex m_txrequest_mutex;
```

Comment:
Renamed.

I have seen nits on other PRs suggesting the exact reverse, which is why i named it this way in the first place, hoping to avoid the nit 😅 But i actually kind of agree that its redundant, so i shouldn't have preemptively tried to avoid the nit.